File: c/1153.c
#include <stdio.h>


int fatorial(int n);

int main(){
    int n, aux=1, i;
    scanf("%d", &n);
    aux = fatorial(n);
    // for(i=1;i<=n;i++){
    //     aux *= i;
    // }
    printf("%d\n", aux);
}

int fatorial(int n){
    if (n<=1){
        return 1;
    }else{
    return n*fatorial(n-1);
    }
}

File: php/PROBLEMS/1153.php
<?php

function fatorial($n){
    if ($n<=1){
        return 1;
    }else{
        return $n * fatorial($n-1);
    }
}

$n = intval(fgets(STDIN));

$aux = fatorial($n);
printf("%d\n", $aux);
?>